forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/rest/TaskFilterRestResource.java

```diff
@@ -18,4 +21,11 @@ public interface TaskFilterRestResource {
     @Produces({MediaType.APPLICATION_JSON, Hal.APPLICATION_HAL_JSON})
     @Consumes(MediaType.APPLICATION_JSON)
     Object queryList(@Context Request request, TaskQueryDto extendingQuery, @QueryParam("firstResult") Integer firstResult, @QueryParam("maxResults") Integer maxResults) throws JsonProcessingException;
+
+    @POST
```

Comment:
Can we add an endpoint to get the count for all filters too? So it will accept list of TaskQueryDto and returns with each count populated. This would avoid multiple calls to BPM.